src/server/server.cpp

```diff
@@ -2869,10 +2871,13 @@ void server::delete_game(int gameid)
 	}
 
 	// send users in the game a notification to leave the game since it has ended
-	static simple_wml::document leave_game_doc("[leave_game]\n[/leave_game]\n", simple_wml::INIT_COMPRESSED);
+	simple_wml::document leave_game_doc_reason("[leave_game]\n[/leave_game]\n", simple_wml::INIT_COMPRESSED);
```

Comment:
Not sure if it makes much of a difference but I would keep the static variable for the common "no reason" case and use another document (without `INIT_COMPRESSED`) for the case with a dynamic reason.

Reply:
Creating the document as:
`simple_wml::document leave_game_doc_reason(simple_wml::string_span("[leave_game]\n[/leave_game]\n"));`
results in the server dying with:
`error server: terminated by C++ exception: failed to uncompress`
when I provide a reason to the stopgame command.

Reply:
That calls a different constructor. I meant using INIT_STATIC instead.

Reply:
Ah, alright, updated.